packages/compass-crud/src/components/json-editor.jsx

```diff
@@ -244,36 +238,26 @@ class EditableJson extends React.Component {
    * @returns {Component} The footer component.
    */
   renderJson() {
-    const OPTIONS = {
-      tabSize: 2,
-      fontSize: 11,
+    const options = {
       minLines: 2,
-      maxLines: Infinity,
-      showGutter: true,
-      readOnly: !this.state.editing,
       highlightActiveLine: false,
       highlightGutterLine: false,
       showLineNumbers: this.state.editing,
-      vScrollBarAlwaysVisible: false,
-      hScrollBarAlwaysVisible: false,
       fixedWidthGutter: false,
       showPrintMargin: false,
       displayIndentGuides: false,
       wrapBehavioursEnabled: true,
-      foldStyle: 'markbegin',
-      useWorker: false
+      foldStyle: 'markbegin'
```

Comment:
I'm not seeing this fold style show up, maybe we don't pass it down? Can look more tmrw/Monday also.
(top is this branch)
<img width="433" alt="Screen Shot 2022-04-28 at 3 24 25 AM" src="https://user-images.githubusercontent.com/1791149/165699495-037cd02e-8c12-48cd-9189-2f0f46fa9f28.png">